npm/nuka-carousel/4.7/@n4jsd/nuka-carousel/index.n4jsd

```diff
@@ -134,6 +134,8 @@ export external public interface ~NukaCarouselSlideRenderControlProps {
     public wrapAround: boolean;
 }
 
+export external public type RenderControlFn = (props: NukaCarouselSlideRenderControlProps) => React.Element<?> | constructor{React.Component};
```

Comment:
Although it makes naming longer, I strongly opt for using prefixes (NukaCarousel in this case) for all types to browse them properly.